spynnaker_integration_tests/test_various/test_send_multiple_partitions.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+
+# Copyright (c) 2024 The University of Manchester
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from time import sleep
+import numpy
+from spinnaker_testbase import BaseTestCase
+import pyNN.spiNNaker as sim
+
+
+class TestSendMultiplePartitions(BaseTestCase):
+
+    def send_spike(self, label, conn):
+        sleep(0.1)
+        conn.send_spike(label, 0)
+
+    # Added to check that the delay expander runs; a previous fix
+    # for a related issue inadvertently turned it off for this type of case
+    def do_run(self):
+        conn = sim.external_devices.SpynnakerLiveSpikesConnection(
+            send_labels=["Inject"], local_port=None)
+        conn.add_start_callback("Inject", self.send_spike)
```

Comment:
he method 'add_start_callback(label, start_callback)' is in deprecation, and will be replaced with the method 'add_start_resume_callback(label, start_resume_callback)' in a future release.